cmd/snap/cmd_list_test.go

```diff
@@ -28,6 +28,26 @@ import (
 	snap "github.com/snapcore/snapd/cmd/snap"
 )
 
+func (s *SnapSuite) TestListHelp(c *check.C) {
```

Comment:
It doesn't seem so great to have a copy of every single help text output just to make sure we won't change it without changing it twice, even more when it doesn't solve the actual problem this PR fixes it:we missed a flag altogether, and can miss again on the future.

A better test would be to programmatically ensure that all the flags registered have proper documentation for them. That will prevent the actual bug from happening again.